src/weights.rs

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2021 Integritee AG
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+//! Autogenerated weights for pallet_teerex
```

Comment:
can you please comment the setup you used to run the benchmark? CPU, RAM and HDD type?

Reply:
Done, it will update the template in the integritee-parachain to add placeholders for these fields. 

FYI, we will only use the trait definition from this pallet and implement it again in the integritee-parachain-runtime. This is the approach that substrate also uses, which simplifies weight updates.

I am a little unsure if we should remove the `IntegriteeWeights` implementation here.